Find a functionally equivalent code:android.animation.AnimatorSet.Builder.after:COMMENT
Method Modifier: public      
Comment:/**
 * Sets up the given animation to play when the animation supplied in the
 * {@link AnimatorSet#play(Animator)} call that created this <code>Builder</code> object
 * to start when the animation supplied in this method call ends.
 *
 * @param anim The animation whose end will cause the animation supplied to the
 * {@link AnimatorSet#play(Animator)} method to play.
 */

@@ -1,4 +1,5 @@
 {
+    mReversible = false;
     Node node = mNodeMap.get(anim);
     if (node == null) {
         node = new Node(anim);

